src/components/ImageCard/ImageCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { H2, Li, Span, Ul } from './ImageCard.styles';
+
+function ImageCard(props) {
+  const RenderMedia = () => {
+    if(props.isVideo === "video"){
+      return (
+        <iframe
+            id="ytplayer"
+            title="watchVideoIframe"
+            type="text/html"
+            width="100%"
+            height="100%"
+            src={props.media}
+            frameBorder="0"
+            allowFullScreen
+          />
+      );
+    }else{
+       return (
+        <img src={props.media} alt="" />
+      );
+    }
+  }
+
+  return (
+    <Ul
+      onClick={props.onClick}
+      mini={props.mini}
+    >
+      <Li className="videoImg">
+        <RenderMedia />
+      </Li>
+      <Li className="videoTitle">
+        <H2 className="" mini>
+          {props.title}
+        </H2>
+      </Li>
+      {!props.mini ? (
+        <>
+          <Li className="videoDescr">
+            <Span className="">{props.explanation}</Span>
+          </Li>
+          <Li className="videoChannel">
+            <Span className="">{props.date}</Span>
+          </Li>
+        </>
+      ) : null}
```

Comment:
Este es también un comentario que dejé en el proyecto pasado, puede que se te haya pasado, entonces lo repetiré, puedes evitar el uso de ternarios y mejorar la lectura si utilizas el operador `&&` (AND), eso ya hará que si el valor que utilizas como condición es `true` regrese lo que hay después del operador, en este caso tu fragment. Se logra lo mismo con menos caracteres.

```suggestion
      {!props.mini &&
        <>
          <Li className="videoDescr">
            <Span className="">{props.explanation}</Span>
          </Li>
          <Li className="videoChannel">
            <Span className="">{props.date}</Span>
          </Li>
        </>
      )}
```